yt_dlp/extractor/trovo.py

```diff
@@ -43,7 +43,7 @@ def _extract_streamer_info(self, data):
 
 
 class TrovoIE(TrovoBaseIE):
```

Comment:
is it possible to add tests for this?

Reply:
This extractor is only for live streams, so every test, except for only matching tests, would fail after the live stream ends.

Reply:
ah cool. Makes sense why there were no tests to begin with.